### DISCUSSION TITLE
[BUG] CLI silently ignoring duplicates in trash

### DISCUSSION THREAD

**Discussion by helgehatt** [immich-app/immich#14346] [Feature Request]
### The bug

Spent half an hour trying to figure out why the photos I uploaded weren't added to the specified album.

Running `immich upload -A "asdf" *` 48/48 photos were identified as duplicates, which was expected, but none of them were added to the album "asdf". It remained empty, which was unexpected.

Turns out I had forgotten to permanently delete the assets from trash before re-uploading.

I would expect either
1. The photos to be restored from trash and added to the album.
2. The CLI to complain that it cannot add trashed assets to the album.

### The OS that Immich Server is running on

Ubuntu 22.04.5

### Version of Immich Server

v1.120.2

### Version of Immich Mobile App

N/A

### Platform with the issue

- [X] Server
- [ ] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
N/A
```

### Your .env content

```Shell
N/A
```

### Reproduction steps

1. Move asset to trash (without permanently deleting it).
2. Try to add that asset to an album using the cli. 
3. The asset is not added to the album nor does it show any errors.
...

**Comment by mmomjian** [comment]
Thanks for the report. This is the expected behavior. #12521

Converting to a feature request in case we want to add a feature to also return that info to the CLI
